crates/forge/Cargo.toml

```diff
@@ -68,6 +68,7 @@ url.workspace = true
 fs_extra.workspace = true
 project-root.workspace = true
 indoc.workspace = true
+indicatif.workspace = true
```

Comment:
I guess this is not needed for this change

Reply:
Removed.